scoper.inc.php

```diff
@@ -93,6 +94,7 @@
 
 		// This dependency is a global function which should remain global.
 		'vendor/ralouphie/getallheaders/src/getallheaders.php',
+		'vendor/symfony/polyfill-intl-normalizer/bootstrap.php',
```

Comment:
There may be others that are needed here, this raised a failure without it so it was the only one added. See the generated `third-party/vendor/autoload_files.php` for other possible entries here.